crawl-ref/source/describe.cc

```diff
@@ -2926,6 +2927,26 @@ void describe_spell(spell_type spelled, const monster_info *mon_owner,
     }
 }
 
+/**
+ * Examine a given ability. List its description and details.
+ *
+ * @param ability   The ability in question.
+ */
+void describe_ability(ability_type ability)
+{
+#ifdef USE_TILE_WEB
+    // XXX I have no idea what show_as_menu() does, can't find definition
+    tiles_crt_control show_as_menu(CRT_MENU, "describe_ability");
+#endif
+
+    string desc = get_ability_desc(ability);
+    print_description(desc);
```

Comment:
Inlining the `desc` variable looks clearer to me.

Reply:
I agree. However, this is my first time doing a pull request - do I have to inline this in my fork and submit another pull request, or are you going to inline it yourself?

Reply:
You can make the change yourself in the same branch and push, which will cause the PR to update; and then (optionally) rebase so it's all one commit. (If you don't do that someone will probably eventually do it for you.)